scripts/create-kubestellar-demo-env.sh

```diff
@@ -134,14 +97,85 @@ checking_cluster() {
             
         else
             echo "$1 not found. Trying again..."
-            sleep 5
+            sleep 20
         fi
 
     done
 }
+##########################################
+
+echo -e "\nStarting environment clean up..."
+echo -e "Starting cluster clean up..."
+
+yq -i 'del(.preferences)' ${KUBECONFIG:-$HOME/.kube/config} 
+
+cluster_clean_up "kind delete cluster --name kubeflex" &
+cluster_clean_up "kind delete cluster --name cluster1" &
+cluster_clean_up "kind delete cluster --name cluster2" &
+wait
+echo -e "Cluster space clean up has been completed"
+
+echo -e "\nStarting context clean up..."
+context_clean_up
+echo "Context space clean up completed"
+
+echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
+clusters=(cluster1 cluster2)
+for cluster in "${clusters[@]}"; do
+   (
+     echo -e "Creating cluster ${cluster}..."
+     kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
+     kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
```

Comment:
I would not run two file-modifying commands concurrently.

Reply:
Benjamin Elder replied in https://kubernetes.slack.com/archives/C2GL57FJ4/p1729873069405909, saying that `kubectl config` _does_ serialize its modifications to your kubeconfig file.